apps/chameleon/components/email-template.tsx

```diff
@@ -1,18 +1,105 @@
 import * as React from 'react'
+import {
+  Body,
+  Button,
+  Container,
+  Head,
+  Heading,
+  Hr,
+  Html,
+  Img,
+  Link,
+  Preview,
+  Section,
+  Text,
+} from '@react-email/components'
+import { Tailwind } from '@react-email/tailwind'
 
 interface EmailTemplateProps {
-  emails: string[]
   name: string
   message: string
-  subject: string
 }
 
+//baseUrl is used to get the logo image <- Warning: local image path currently for demo purposes
+const baseUrl = process.env.VERCEL_URL
```

Comment:
You need to use the helper function available in https://github.com/Ionia-Devs/chameleon/blob/35fe7b10ce3a71cf4a304a3f54d91d1485ba4234/apps/chameleon/lib/utils.ts#L19-L21